grammers-client/src/types/message.rs

```diff
@@ -231,7 +231,7 @@ impl Message {
         self.raw.id()
     }
 
-    pub(crate) fn peer_ref(&self) -> PeerRef {
+    pub fn peer_ref(&self) -> PeerRef {
```

Comment:
Yeah fair, I wanted to revisit these and make sure they were consistent at some point.

I think we'll want `peer` and `peer_ref` on pretty much everything that contains a peer (and sometimes `sender` and `sender_ref`).